app/api/dao/admin.py

```diff
@@ -1,10 +1,11 @@
 from app.database.models.user import UserModel
-
+from app.utils.decorator_utils import email_verification_required
 
 class AdminDAO:
     """Data Access Object for Admin functionalities."""
 
     @staticmethod
+    @email_verification_required
     def assign_new_user(assigner_user_id, data):
```

Comment:
```suggestion
    def assign_new_user(user_id, data):
```

Reply:
don't forget to fix the doc strings to reflect the new argument name

Reply:
Done